app/routes/joblistings/components/JoblistingList.css

```diff
@@ -63,43 +60,36 @@
   height: 80px;
   min-width: 120px;
   object-fit: contain;
+  margin-right: 20px;
 }
 
 .listItem {
+  display: flex;
   flex: 1;
   justify-content: space-between;
+  font-size: 14px;
```

Comment:
I may just be getting old, but I thought this made the text a little to small

Reply:
I definitely see your point. I think it stands out a bit much since the rest of the text on the page is quite larger. In general we use very large font sizes on the webapp, which I don't really like. Nevertheless, I bumped it up to `15px`, and the title to `16px`. It now matches the rest, so I hope that's not too small he he